doc/paradox/sql-to-slick.md

```diff
@@ -155,7 +155,7 @@ Scala's equivalent for `SELECT` is `map`. Columns can be referenced similarly an
 @@snip [SqlToSlick.scala](../code/SqlToSlick.scala) { #sqlQueryFilter }
 
 <h4>Slick</h4>
-Scala's equivalent for `WHERE` is `filter`. Make sure to use `===` instead of `==` for comparison.
+Scala's equivalent for `WHERE` is `filter`. Make sure to use `===` instead of `==` for comparison. Slick also preserves the SQL behavior throughout filter-like methods (`filter`, `filterIf`, `filterOpt`, `withFilter`), where the expressions involving `NULL` (corresponding to `None` in Scala/Slick) yield `NULL`, hence evaluated as `false` and such rows are filtered out. 
```

Comment:
```suggestion
Scala's equivalent for `WHERE` is `filter`. There are also the convenience variations `filterIf` and `filterOpt`.

@@@ note

In Slick, `Option` types correspond to nullability in SQL. If a filter predicate has type `Rep[Option[Boolean]]` that means the database can evaluate it to `TRUE`, `FALSE`, or `NULL`, but `NULL` behaves like `FALSE`. This behavior is consistent across `filter`, `filterIf`, `filterOpt`, and `withFilter`.

@@@

@@@ warning

Make sure to use `===` instead of `==` for comparison, so that Slick can convert the expression to an SQL `=`. Slick uses `===` for that because `==` is already used by Scala for performing comparisons locally.

@@@
```

Reply:
Much better, thank you.